build-a-technical-documentation-page.txt

```diff
@@ -0,0 +1,444 @@
+** start of undefined **
+
+<!DOCTYPE html>
+<html>
+<head>

```

Comment:
[Optional] Here in the Header, you could put the nav bar, so screen readers have a better time.